### PR TITLE
Event Aggregator Record

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#63149](https://central.tri.be/issues/63149)

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $update is never used | [Record.php:182](https://github.com/moderntribe/the-events-calendar/blob/9e331009d278e18c1e70d86d1a13a18c2df851e0/src/Tribe/Aggregator/Record.php#L182)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Aggregator/Record.php [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+	public function action_do_import() {
+		return wp_send_json_success();
+	}
+
+	public function filter_maybe_empty_content( $maybe_empty = false, $postarr = array() ) {
+
+	}
+
+	public function action_check_scheduled_imports( $post_ID, $post, $update ) {
+		var_dump( $post, $post_ID );
```
Comment (barryhughes) [inline src/Tribe/Aggregator/Record.php]: Leftover from testing?

Comment (bordoni) [inline src/Tribe/Aggregator/Record.php]: My bad, this one is on Hold. Still things to be done here.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $origin is never used | [Record.php:184](https://github.com/moderntribe/the-events-calendar/blob/fc6da77b6813a174f9826c819cc537692e6010e1/src/Tribe/Aggregator/Record.php#L184)
:x: | Empty array declaration must have no space between the parentheses | [Record.php:195](https://github.com/moderntribe/the-events-calendar/blob/fc6da77b6813a174f9826c819cc537692e6010e1/src/Tribe/Aggregator/Record.php#L195)
:warning: | Comment refers to a TODO task | [Record.php:210](https://github.com/moderntribe/the-events-calendar/blob/fc6da77b6813a174f9826c819cc537692e6010e1/src/Tribe/Aggregator/Record.php#L210)

> via jenkins/codesniffer-phpcs

**Comment by bordoni** [comment]
Me and @borkweb decided to merge this one, because we are going to Push a new Pull Request To Event Aggregator with the final touches to this part of code.

**Pull request merged**